docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -100,21 +99,28 @@ Note: Before you go on annual leave, tell the delivery manager who will be looki
 
   You will now be prompted to continue or cancel.
 
-9. Raise a pull request, with `support/<CURRENT MAJOR VERSION NUMBER>.x` as the base branch to merge into.
+9. Raise a pull request, with `support/<MAJOR VERSION NUMBER>.x` as the base branch to merge into.
 
-10. Once a developer approves the pull request, merge it into `support/<CURRENT MAJOR VERSION NUMBER>.x`.
+10. Once a developer approves the pull request, merge it into `support/<MAJOR VERSION NUMBER>.x`.
 
 ### Publish the release to npm
 
-1. Check out `support/<CURRENT MAJOR VERSION NUMBER>.x` and pull the latest changes.
+1. Check out `support/<MAJOR VERSION NUMBER>.x` and pull the latest changes.
 
 2. Sign in to npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
 
-3. Run `npm run publish-release`, which will prompt you to either continue or cancel. Enter `y` to continue.
+3. Run `npm run publish-release`, which will prompt you to check whether the npm tag looks as expected.
+
+  If you're following these instructions, you may be publishing an out-of-order release. Do
```

Comment:
Just looked at the [Slack convo](https://gds.slack.com/archives/CAF8HTJES/p1653049737575209)...

Can see the logic for each of the suggestions, but I'd probably lean towards 'a release for the current major version', if that's appropriate. I think with each of the others, a new user of this content might wonder "What does 'backport'/'maintenance release'/'support release' mean?"

Reply:
@EoinShaughnessy I've just had a brainwave and realised that actually I think the only time someone may not want to tag as "latest" is when they're doing a release for a previous major version... I've adjusted the text to reflect this.